in_toto/cmd/verify.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	intoto "github.com/boxboat/in-toto-golang/in_toto"
+	"github.com/spf13/cobra"
+)
+
+var verifyCmd = &cobra.Command{
+	Use:   "verify",
+	Short: "Verify that the software supply chain of the delivered product",
+	Long: `in-toto-verify is the main verification tool of the suite, and 
+it is used to verify that the software supply chain of the delivered 
+product was carried out as defined in the passed in-toto supply chain 
+layout. Evidence for supply chain steps must be available in the form 
+of link metadata files named ‘<step name>.<functionary keyid prefix>.link’.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		var layoutMb intoto.Metablock
+
+		if err := layoutMb.Load(layoutPath); err != nil {
+			fmt.Println(err.Error())
+		}
+
+		//Load Keys
+		layoutKeys := make(map[string]intoto.Key, len(pubKeyPaths))
+
+		for _, pubKeyPath := range pubKeyPaths {
+			var pubKey intoto.Key
+
+			if err := pubKey.LoadKey(pubKeyPath, "rsassa-pss-sha256", []string{"sha256", "sha512"}); err != nil {
```

Comment:
These are reasonable default, but I've run into this elsewhere.  What if we have an ed25519 key?  Seems like we need a way to load a key without knowing it's type ahead of time.

We can kick that can down the road, though.